inventory.d/macmini-r8.yaml

```diff
@@ -195,7 +195,17 @@ groups:
       - macmini-r8-249.test.releng.mdc1.mozilla.com
       - macmini-r8-251.test.releng.mdc1.mozilla.com
       - macmini-r8-253.test.releng.mdc1.mozilla.com
+      - macmini-r8-256.test.releng.mdc1.mozilla.com
       - macmini-r8-257.test.releng.mdc1.mozilla.com
+      - macmini-r8-258.test.releng.mdc1.mozilla.com
+      - macmini-r8-259.test.releng.mdc1.mozilla.com
+      - macmini-r8-260.test.releng.mdc1.mozilla.com
+      - macmini-r8-261.test.releng.mdc1.mozilla.com
+      - macmini-r8-262.test.releng.mdc1.mozilla.com
+      - macmini-r8-263.test.releng.mdc1.mozilla.com
+      - macmini-r8-264.test.releng.mdc1.mozilla.com
+      - macmini-r8-265.test.releng.mdc1.mozilla.com
+      - macmini-r8-266.test.releng.mdc1.mozilla.com
```

Comment:
I just merged the other PR, so I think these nodes are already added now at this point